raiden/storage/sqlite.py

```diff
@@ -1474,6 +1488,18 @@ def update_light_client_protocol_message_set_signed_data(self, payment_id, msg_o
             last_id = cursor.lastrowid
         return last_id
 
+    def update_light_client_protocol_message_with_signed_message(self,
```

Comment:
We already have a `update_light_client_protocol_message_set_signed_data ` method

The only difference is that this one receives the msg order and type. 

Maybe we can rename this new method to `update_light_client_protocol_message_set_signed_data_by_id`

Or just keep one of them.

Reply:
Probably we will need to remove the older method or rename this one but it will depend on the path we want to take from now to do those updates, actually renaming this to `update_light_client_protocol_message_set_signed_data_by_id` could be a problem since which id? the internal id like this one does or the message_identifier like we use in other places? that's why i didn't make any refactor on that, i agree that we should have only one method for this but we don't have the internal identifier on the cases that are being used right now.

Reply:
The update_light_client_protocol_message_set_signed_data  is related with a payment because is for offchain msgs

The new method update_light_client_protocol_message_with_signed_message isnt always related with a payment (register secret is but unlock and settlement not) 

Maybe we can rename both methods to

`update_ofchain_light_client_protocol_message_set_signed_data(payment,order, type,msg)` and `update_onchain_light_client_protocol_message_set_signed_data(id)`

what do u guys think? 

@mauricioirace @mortelli @JONAF2103

Reply:
I agree with your proposal @marcosmartinez7 
Maybe we could use something like:
`update_offchain_light_client_protocol_message_set_signed_message` and `update_onchain_light_client_protocol_message_set_signed_transaction` to tell the function caller what should it send as signed data. But any of the two approaches seems fine to me.

Reply:
Agree!